### ISSUE TITLE
Abstract decks

### ISSUE THREAD

**Issue by RafaelGCruvinel** [devleaks/streamdecks#1]
Hi!

I was searching for a solution like [python-elgato-streamdeck](https://github.com/abcminiuser/python-elgato-streamdeck),
but for Loupedeck. And this repo is the only one I found..
Are you changing the behavior just for this repo proposal or were you inspired by another project??
If you approve us to decoup the Loupedeck class, I can help this project evolving loupedecks features.

Thanx!

**Comment by devleaks** [comment]
Hi.
I was "inspired" by that [python-elgato-streamdeck](https://github.com/abcminiuser/python-elgato-streamdeck) project  for organisation of files and classes. So I badly and very quickly did this [python-loupedeck-live](https://github.com/devleaks/python-loupedeck-live) project. (You couldn't see the project, it was private, I just published it right now.) Let me know if it helps.
There are numerous things that were not completed. it is work in progress, but feel free to contribute.
I'm not a "hardware" guy, so I don't know much about USB HID, or low level access through serial connections.
I got a thing working and that's what matters. Someone else, more competent in those area, can polish the code.
P.
